src/MICore/Transports/LocalLinuxTransport.cs

```diff
@@ -187,6 +195,15 @@ public override void InitStreams(LaunchOptions options, out StreamReader reader,
             reader = new StreamReader(gdbStdOutStream);
         }
 
+        private void FifoWatcher_Deleted(object sender, FileSystemEventArgs e)
+        {
+            if (e.FullPath.Contains(_gdbStdInName) || e.FullPath.Contains(_gdbStdOutName))
+            {
+                _fifoWatcher.Deleted -= FifoWatcher_Deleted;
+                this.Callback.OnDebuggerProcessExit(null);
```

Comment:
Since the fifos are always deleted on debugging stop, does this now mean we're trigging exit twice?